test/helpers.go

```diff
@@ -0,0 +1,94 @@
+package test
```

Comment:
is `t.Skip()` a suitable way of limiting the test? I'm also ok with either removing the os-specific tests, or moving them to an integration test suite